package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "misty-config",
+  "version": "1.0.0",
+  "description": "Cascading configuration with environment variable support",
+  "main": "index.js",
+  "scripts": {
+  },
+  "author": "Body Labs Inc.",
+  "license": "MPL-2.0",
```

Comment:
Any reason for picking this license? Our standard license is in here:

https://github.com/bodylabs/skeleton-library

We tend to use other licenses when there is an open-source ecosystem argument.